### PR TITLE
INFRA: Initialize Unit Test

### PR DESCRIPTION
**Pull request by ADILATIC**

### CODE REVIEW

File: InternTrack.Core.Unit.Tests/InternTrack.Core.Unit.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```
Comment (hassanhabib) [inline InternTrack.Core.Unit.Tests/InternTrack.Core.Unit.Tests.csproj:6]: Both of these need to be disabled.

File: .github/workflows/dotnet-desktop.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+# built on .NET Core.
+# To learn how to migrate your existing application to .NET Core,
+# refer to https://docs.microsoft.com/en-us/dotnet/desktop-wpf/migration/convert-project-from-net-framework
+#
+# To configure this workflow:
+#
+# 1. Configure environment variables
+# GitHub sets default environment variables for every workflow run.
+# Replace the variables relative to your project in the "env" section below.
+#
```
Comment (hassanhabib) [inline .github/workflows/dotnet-desktop.yml:16]: Why do we need this file?

**Comment by hassanhabib** [comment]
@ADILATIC where's the example test file? Like in the video?

**Comment by glhays** [comment]
@ADILATIC @hassanhabib This PR needs closed in lieu of PR #13

**Pull request closed without merging**